CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 CHANGELOG
 =========
 
+1.0.3
+=====
+
+* bug: Move processing of requirements file out to the specific container.
```

Comment:
can you match the style we've done for the Python SDK where there's a 1-2 word indicator for what part of the repo is being changed? e.g. `bug: TrainingEnvironment: Move...`